### ISSUE TITLE
[docker] Porting to Debian 11 (stable)

### ISSUE THREAD

**Issue by mmaridev** [CollaboraOnline/online#4695]
**This is an [Easy Hack](https://collaboraonline.github.io/post/easyhacks/).**
Potential mentors: @thebearon @timar @mmaridev 

### Detailed description and rationale
Update the Debian dockerfile to Debian 11. Some packages were renoved / renamed.

### Code pointers
https://github.com/CollaboraOnline/online/blob/master/docker/from-source/Debian

### Links to related documentation
#3662

### Additional notes, tips & tricks
Try changing the `FROM` and run the docker build. See what packages are no longer available in bullseye and investigate.

**Comment by callmetushar123** [comment]
Hello @mmaridev , If I am not wrong, once I see the packages that are no longer available, I will need to update the docker file to install them right?

**Comment by mmaridev** [comment]
Basically, yes. Please, do not just go ahead and remove all the missing ones: try to understand why they are no longer available in bullseye and what was their function within the project.

If you need further help, you can start by posting here the list of packages no longer to be found and we can discuss together.

**Comment by mmaridev** [comment]
You'll need to ask someone that is maintainer of the project 
/cc @thebearon @timar

**Comment by callmetushar123** [comment]
@pedropintosilva  assign me here too. I would like to take up this issue too.

**Comment by callmetushar123** [comment]
@mmaridev 

![dockerbuild](https://user-images.githubusercontent.com/31562363/194710121-93ed992e-7335-483d-8b7c-f960c493c4bd.jpg)

So I could successfully build Debian in my windows.
![debian1newpackages](https://user-images.githubusercontent.com/31562363/194710177-586d0fd0-9071-49e1-bac4-d181b300abbd.jpg)
These were the new packages that were installed
![debian1fail](https://user-images.githubusercontent.com/31562363/194710201-714b2183-ef99-4c37-9dd0-afc5e33580c7.jpg)
libubsan0 was deprecated, so I have removed it from the list of packages to be installed.

PS I ran into a server problem so added the line `RUN apt-get clean && apt-get update` and the build was completed without any errors.
